README.md

```diff
@@ -12,6 +12,8 @@ Some targets and devices are maintained by GHI Electronics while others are main
 - G30
 - G80
 - USBizi
+- [WiFiMCU](http://www.wifimcu.com/)
```

Comment:
These two links shouldn't be under the list officially maintained by us. I'd create a separate section after the one talking about official GHI devices for community devices and put these two there.

Reply:
I see.
I'll rollback README.md.